ts/components/LabelledItem.tsx

```diff
@@ -27,20 +27,24 @@ const styles = StyleSheet.create({
   }
 });
 
+type StyleType = { [key in string]: string | number };
+
 type Props =
```

Comment:
to make the code more readable you could change this type definition as follow:

```typescript
type CommonProp = Readonly<{
  label: string;
  icon: string;
  isValid?: boolean;
  iconStyle?: StyleType;
}>;
type Props = CommonProp &
  (
    | Readonly<{
        type: "masked";
        inputMaskProps: TextInputMaskProps;
      }>
    | Readonly<{
        type: "text";
        inputProps: TextInputProps;
      }>);
```

Reply:
changed i n 0827dd5b